src/Packages/UnityWebBrowser/Runtime/Core/RawImageUwbClientInputHandler.cs

```diff
@@ -89,42 +101,54 @@ private IEnumerator KeyboardAndMouseHandler()
 
             while (Application.isPlaying)
             {
-                if(!browserClient.IsConnected)
+                yield return 0;
+
+                if (!browserClient.IsConnected)
+                    continue;
+
+                if (disableMouseInputs && disableKeyboardInputs)
                     continue;
-                
+
                 if (GetMousePosition(out Vector2 pos))
                 {
-                    //Mouse position
-                    if (lastSuccessfulMousePositionSent != pos)
+                    if (!disableMouseInputs)
                     {
-                        browserClient.SendMouseMove(pos);
-                        lastSuccessfulMousePositionSent = pos;
+                        //Mouse position
+                        if (lastSuccessfulMousePositionSent != pos)
+                        {
+                            browserClient.SendMouseMove(pos);
+                            lastSuccessfulMousePositionSent = pos;
+                        }
+
+                        //Mouse scroll
+                        float scroll = inputHandler.GetScroll();
+                        scroll *= browserClient.BrowserTexture.height;
+
+                        if (!disableMouseInputs && scroll != 0)
```

Comment:
Don't need this extra `!disableMouseInputs` check.

Reply:
Done!